docs/semgrep-secrets/getting-started.md

```diff
@@ -1,153 +1,139 @@
+| Confirmed valid | Semgrep made an API call using the secret and it returned an HTTP response of 200 or similar and granted authentication. |
+| Confirmed invalid | Semgrep made an API call using the secret and it returned an HTTP response of 403 or similar. |
+| Validation error | Semgrep made an API call but it returned and HTTP response of 400 or similar; a server error, such as a timeout, occurred. The Semgrep Team recommends manually reviewing the finding. |
+| No validator | Semgrep does not perform any validation on this finding. You must manually review the finding. |
 
-## Receiving findings in GitHub through PR comments
+### Repository visibility
 
-This section describes how to set up PR or MR comments from Semgrep. 
+Refers to whether or not the repository is a public repository or private. This is detected through your source code manager.
 
 :::info
-After enabling this feature, only **valid** (active) secrets-related findings leave a PR or MR comment.
+Semgrep supports visibility detection only for GitHub repositories of any plan.
 :::
 
-![Semgrep Secrets finding in a PR comment](/img/secrets-pr-comment.png#bordered)
-**_Figure._** Semgrep Secrets finding in a PR comment.
+| Repository visibility | Description | 
+| -----------  | ------------ |
+| Public | Repository access doesn't require authentication; at a minimum, it can be viewed by anyone. |
+| Private | Repository access requires authentication. |
+| Unknown | Semgrep Secrets is unable to detect your repository visibility. This is typically assigned to: <ul><li>Scans from local developer machines.</li><li>Scans from any non-GitHub source code manager, such as GitLab.</li></ul> |
 
-### Findings in GitHub pull requests
+### Type
 
-Perform the following steps to receive Secrets findings as comments in GitHub PRs:
+Refers to the type of secret, such as **private key**, or the web service that makes use of the secret, such as **Sendgrid** or **Stripe**.
 
-1. Follow the steps in [<i class="fa-regular fa-file-lines"></i> GitHub PR comments](/semgrep-cloud-platform/github-pr-comments/).
-2. Inform [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com) that you want to enable this feature.
+### Project
 
-<!--### Findings in GitLab merge requests
+A repository that's been onboarded to Semgrep for scanning.
 
-Perform the following steps to receive Secrets findings as comments in GitLab MRs:
+### Branch
 
-1. Follow the steps in [<i class="fa-regular fa-file-lines"></i> GitLab MR comments](/semgrep-cloud-platform/gitlab-mr-comments/).
-2. Inform [<i class="fa-regular fa-envelope"></i> support@semgrep.com](mailto:support@semgrep.com) that you want to enable this feature.
--->
-## Triaging secrets in Semgrep Cloud Platform
+A branch of any Project.
 
-Triage secrets-related findings in the Secrets page. By default, all findings are displayed. A common triage workflow includes the following tasks:
+## Triage secrets in Semgrep Cloud Platform
```

Comment:
I wonder if this should already be its own document? This intro/getting started is quite long.